### DISCUSSION TITLE
Status of EKS 1.22 in terraform-aws-service-catalog

### DISCUSSION THREAD

**Discussion by BrianMusson** [gruntwork-io/knowledge-base#358] [Help] [answered]
Hi, AWS released EKS 1.22. I wanted to see if there has been testing with the current catalog or when it will become the new default. Thanks!

https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-1.22
[r:terraform-aws-eks](https://github.com/gruntwork-io/terraform-aws-eks)
[r:terraform-aws-service-catalog](https://github.com/gruntwork-io/terraform-aws-service-catalog)

**Comment by autero1** [comment, marked as answer]
Hi! We'll track progress in https://github.com/gruntwork-io/terraform-aws-eks/issues/430
